python/GafferUI/PathListingWidget.py

```diff
@@ -620,7 +620,7 @@ def __buttonPress( self, widget, event ) :
 
 		# Otherwise, perform expansion and selection as appropriate.
 
-		if event.buttons != event.Buttons.Left :
+		if event.buttons not in ( event.Buttons.Left, event.Buttons.Right ) :
```

Comment:
It feels a bit odd that we can accept the right-click here but that the context menu signal still gets emitted. Is that consistent on all platforms (I think context menu signal has some per-platform differences)?

Reply:
Ugh. This behaves reasonably on Linux and macOS, but I gave it a try on Windows and as you suspected the combination is sub-optimal. Right-clicking within a range of already selected items results in the selection updating to just the item clicked. As this behaviour isn't crucial to this PR maybe I'll drop this for now and we can revisit later rather than holding the rest up.